build/commands/lib/util.js

```diff
@@ -490,6 +490,24 @@ const util = {
     util.touchOverriddenVectorIconFiles()
   },
 
+  touchGsutilChangeLogFile: () => {
```

Comment:
what happens if we don't do this? There is no information at all in the follow-up ticket

Reply:
otherwise we get error on running tests, added error sample into the follow up ticket